openerp/addons/openupgrade_records/model/analysis_wizard.py

```diff
@@ -55,14 +55,14 @@ class openupgrade_analysis_wizard(TransientModel):
             [('init', 'Init'), ('ready', 'Ready')], 'State',
             readonly=True),
         'log': fields.text('Log'),
-        'write': fields.boolean(
+        'write_files': fields.boolean(
             'Write files',
             help='Write analysis files to the module directories'
             ),
         }
     _defaults = {
         'state': lambda *a: 'init',
-        'write': lambda *a: True,
+        'write_files': lambda *a: True,
```

Comment:
Sure, for both fields, but I just made the minimum amount of change to fix the issue.